README.md

```diff
@@ -2,6 +2,10 @@
 
 This project was bootstrapped with [Nimble React template](https://github.com/nimblehq/react-templates).
 
+# Prerequisites
+
+NodeJS: 18.13.0 LTS
```

Comment:
I saw you had `.tools-version` in this repo. Are you using `asdf` to mange the Node version? If yes, please update that file as well 🙏

Reply:
Sorry, I forgot to reply to your comment the NodeJs version is updated to LTS version in `.tools-version` and `README.md` files